### PR TITLE
[docs] Improve instructions for the configuration of BE. 

### LINKED ISSUE #7619: [Enhancement] Improve instructions for the configuration of BE.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Description

Recently, I read the documents on [https://doris.apache.org](https://doris.apache.org) and found a typo in [this page](https://doris.apache.org/master/zh-CN/installing/install-deploy.html#be-%E9%83%A8%E7%BD%B2).

It seems there is a missing comma between `SSD` and `10`.

Snipaste_2022-01-05_18-12-18

### Solution

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by adonis0147**
## Proposed changes

Close related #7619.

Fix the typo in document.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on (Fix #7619) and described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/administrator-guide/config/be_config.md [hunk 1]
```diff
@@ -1160,8 +1160,8 @@ storage_flood_stage_usage_percent和storage_flood_stage_left_capacity_bytes两
 
   `storage_root_path=/home/disk1/doris.HDD,50;/home/disk2/doris.SSD,10;/home/disk2/doris`
 
-  * /home/disk1/doris.HDD, 50，表示存储限制为50GB, HDD;
-  * /home/disk2/doris.SSD 10， 存储限制为10GB，SSD；
+  * /home/disk1/doris.HDD,50，表示存储限制为50GB，HDD;
+  * /home/disk2/doris.SSD,10，存储限制为10GB，SSD；
```
Comment (jackwener) [inline docs/zh-CN/administrator-guide/config/be_config.md:1164]: It's ok to use Chinese symbol in Chinese.
It's not essential to modify it.

**Comment by jackwener** [comment]
This PR isn't essential

File: docs/zh-CN/administrator-guide/config/be_config.md (hunk 1 above)
Comment (adonis0147) [reply docs/zh-CN/administrator-guide/config/be_config.md:1164]: It doesn't concern the Chinese symbol, please refer the issue #7619.

Comment (jackwener) [inline docs/zh-CN/administrator-guide/config/be_config.md:1164]:
```suggestion
  * /home/disk1/doris.HDD, 50，表示存储限制为50GB, HDD;
  * /home/disk2/doris.SSD, 10，存储限制为10GB, SSD；
```

File: docs/zh-CN/installing/install-deploy.md [hunk 2]
```diff
@@ -178,8 +178,8 @@ doris默认为表名大小写敏感，如有表名大小写不敏感的需求需
 
     **说明**
 
-    - /home/disk1/doris.HDD, 50，表示存储限制为50GB, HDD;
-    - /home/disk2/doris.SSD 10， 存储限制为10GB，SSD；
+    - /home/disk1/doris.HDD,50，表示存储限制为50GB，HDD;
+    - /home/disk2/doris.SSD,10，存储限制为10GB，SSD；
```
Comment (jackwener) [inline docs/zh-CN/installing/install-deploy.md:182]:
```suggestion
    - /home/disk1/doris.HDD, 50，表示存储限制为50GB, HDD;
    - /home/disk2/doris.SSD, 10，存储限制为10GB, SSD；
```

**Review by jackwener — COMMENTED** [review]
It will be ok to apply this suggestions

File: docs/zh-CN/administrator-guide/config/be_config.md (hunk 1 above)
Comment (adonis0147) [reply docs/zh-CN/administrator-guide/config/be_config.md:1164]: These instructions are used to explain the configuration `storage_root_path=/home/disk1/doris.HDD,50;/home/disk2/doris.SSD,10;/home/disk2/doris`.
The configuration is separated into three parts by the semicolon.
1. /home/disk1/doris.HDD,50
2. /home/disk2/doris.SSD,10
3. /home/disk2/doris

Using the original text is more readable. Therefore, I think my modification is better than the suggestion.

File: docs/zh-CN/installing/install-deploy.md (hunk 2 above)
Comment (adonis0147) [reply docs/zh-CN/installing/install-deploy.md:182]: Same as the above.

**Comment by jackwener** [comment]
LGTM

File: docs/zh-CN/installing/install-deploy.md [hunk 3]
```diff
@@ -178,8 +178,8 @@ doris默认为表名大小写敏感，如有表名大小写不敏感的需求需
 
     **说明**
 
-    - /home/disk1/doris.HDD, 50，表示存储限制为50GB, HDD;
-    - /home/disk2/doris.SSD 10， 存储限制为10GB，SSD；
+    - /home/disk1/doris.HDD,50，表示存储限制为50GB，HDD;
```
[the remark this answers is not in the document]
Comment (adonis0147) [reply docs/zh-CN/installing/install-deploy.md:181]: Please review the new patch.

**Review by EmmyMiao87 — DISMISSED** [review]
LGTM

File: docs/en/administrator-guide/config/be_config.md [hunk 4]
```diff
@@ -1149,9 +1149,9 @@ Cache for storage page size
 
     eg.1: `storage_root_path=/home/disk1/doris.HDD,50;/home/disk2/doris.SSD,1;/home/disk2/doris`
 
-    * 1./home/disk1/doris.HDD, capacity limit is 50GB, HDD;
-    * 2./home/disk2/doris.SSD, capacity limit is 1GB, SSD;
-    * 3./home/disk2/doris, capacity limit is disk capacity, HDD(default)
+    * 1./home/disk1/doris.HDD,50 capacity limit is 50GB, HDD;
+    * 2./home/disk2/doris.SSD,1 capacity limit is 1GB, SSD;
+    * 3./home/disk2/doris capacity limit is disk capacity, HDD(default)
```
Comment (jackwener) [inline docs/en/administrator-guide/config/be_config.md:1154]:
```suggestion
    * 1./home/disk1/doris.HDD,50, indicate capacity limit is 50GB, HDD;
    * 2./home/disk2/doris.SSD,1, indicate capacity limit is 1GB, SSD;
    * 3./home/disk2/doris, indicate capacity limit is disk capacity, HDD(default);
```

File: docs/en/installing/install-deploy.md [hunk 5]
```diff
@@ -179,9 +179,9 @@ See the section on `lower_case_table_names` variables in [Variables](../administ
 
     **instructions**
     
-    * 1./home/disk1/doris.HDD, capacity limit is 50GB, HDD;
-    * 2./home/disk2/doris.SSD, capacity limit is 1GB, SSD;
-    * 3./home/disk2/doris, capacity limit is disk capacity, HDD(default)
+    * 1./home/disk1/doris.HDD,50 capacity limit is 50GB, HDD;
+    * 2./home/disk2/doris.SSD,1 capacity limit is 1GB, SSD;
+    * 3./home/disk2/doris capacity limit is disk capacity, HDD(default)
```
Comment (jackwener) [inline docs/en/installing/install-deploy.md:184]:
```suggestion
    * 1./home/disk1/doris.HDD,50, indicate capacity limit is 50GB, HDD;
    * 2./home/disk2/doris.SSD,1, indicate capacity limit is 1GB, SSD;
    * 3./home/disk2/doris capacity, indicate limit is disk capacity, HDD(default);
```

**Review by jackwener — COMMENTED** [review]
It's better use comma than blank.

**Comment by morningman** [comment]
hi @adonis0147 , there are some unresolved comments, PTAL

File: docs/en/administrator-guide/config/be_config.md (hunk 4 above)
Comment (adonis0147) [reply docs/en/administrator-guide/config/be_config.md]: Please review the new patch.

File: docs/en/installing/install-deploy.md (hunk 5 above)
Comment (adonis0147) [reply docs/en/installing/install-deploy.md]: Same as the above.

**Comment by jackwener** [comment]
LGTM
by the way, you can apply suggestion which will save your energy.

**Comment by adonis0147** [comment]
Hi @morningman , all comments are resolved, PTAL.

**Review by morningman — APPROVED** [review]

**Pull request merged**